src/user-service.ts

```diff
@@ -66,7 +66,8 @@ export class UserService implements UserServiceInterface {
       const user = User.fromUserResponse(persistedUser);
       // verify that the cached used matches the user in the cookie
       // otherwise fetch new user info for the cookie'd user
-      const nameMatches = cookieUsername === user.username;
+      const decodedCookie = decodeURIComponent(cookieUsername);
+      const nameMatches = decodedCookie === user.username;
```

Comment:
This fixes the url-encoding issue. The cookie gets stored as `foo%40bar.com` and the response uses `foo@bar.com` so this comparison was always failing.

Reply:
@bfalling Is there a project in the works to update this so we can update this package at the same time?